src/omnicore/omnicore.cpp

```diff
@@ -2271,7 +2271,10 @@ bool mastercore_handler_tx(const CTransaction& tx, int nBlock, unsigned int idx,
         assert(mp_obj.getEncodingClass() != NO_MARKER);
         assert(mp_obj.getSender().empty() == false);
 
-        fFoundTx |= HandleExodusPurchase(tx, nBlock, mp_obj.getSender(), nBlockTime);
+        // extra iteration of the outputs for every transaction, not needed on mainnet after Exodus closed in block 255365
+        if (isNonMainNet() || nBlock <= 255365) {
```

Comment:
We could get rid of the magic number, by using the consensus params (e.g. [like this](https://github.com/zathras-crypto/omnicore/blob/db67169e60533885ef363f62bceec9ac06552b2f/src/omnicore/omnicore.cpp#L518-L520)).